util-core/src/test/scala/com/twitter/util/PromiseTest.scala

```diff
@@ -344,5 +344,7 @@ class PromiseTest extends FunSuite {
 
   test("CoverageChecker") {
     println("[Coverage] - " + CoverageChecker.map.getOrElse("detach", sys.error(s"unexpected key")).mkString("[",", ", "]"))
+    println("[Coverage] - " + CoverageChecker.map.getOrElse("continue", sys.error(s"unexpected key")).mkString("[",", ", "]"))
```

Comment:
Maybe you could add a prefix here so it says `[Coverage] continue - ` so that you can tell which array is which between the two prints? You should simultaneously modify the previous line to say `[Coverage] detach -`